platform/v1alpha1/platformoperators.crd.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    release.openshift.io/feature-gate: TechPreviewNoUpgrade
+    api-approved.openshift.io: https://github.com/openshift/api/pull/1234
+  creationTimestamp: null
+  name: platformoperators.platform.openshift.io
+spec:
+  group: platform.openshift.io
+  names:
+    kind: PlatformOperator
+    listKind: PlatformOperatorList
+    plural: platformoperators
+    shortNames:
+      - platform
+      - platforms
```

Comment:
Shortname are typically an initialism, I worry that this has lost its operator point and that a `platform` could in the future be something in and of itself

Also, we have seen a lot of folks getting amiguous results when using similar or short names which can cause confusing behaviour when kubectl looks these up. I think as this is alpha it's the least prioritised. If someone created a `platform` as part of an OLM operator that was v1beta1 or v1, the short name here would be useless